content/en/docs/releases/supported-releases/index.md

```diff
@@ -18,7 +18,7 @@ on a minor release do not contain backward incompatible changes.
 - [Control Plane/Data Plane Skew](#control-planedata-plane-skew)
 - [Support status of Istio releases](#support-status-of-istio-releases)
 - [Supported releases without known Common Vulnerabilities and Exposures (CVEs)](#supported-releases-without-known-common-vulnerabilities-and-exposures-cves)
-- [Relationship between Istio and Envoy](#supported-envoy-versions)
+- [Supported Envoy Versions](#supported-envoy-versions)
```

Comment:
I like the previous title better, but either is probably fine. I suspect the link was not changed earlier to prevent having to possibly update places which referred to here.

Reply:
The link itself doesn't change, but now the link text matches the header that it links to.

Reply:
I see now that you were just updating the TOC link title to match the section title.